src/snowflake/snowpark/_internal/analyzer/snowflake_plan.py

```diff
@@ -982,6 +992,9 @@ def __init__(
         )
         self.is_ddl_on_temp_object = is_ddl_on_temp_object
 
+    def __repr__(self) -> str:
+        return f"Query({self.sql}, {self.query_id_place_holder}, {self.is_ddl_on_temp_object})"
```

Comment:
Why does this PR need it?

Reply:
It does not, I just found this to be helpful while debugging. Do you not want me to commit this?

Reply:
It should be fine since it's internal. Let's do it